io.go

```diff
@@ -8,20 +8,22 @@ import (
 
 // Pin represents a single pin, which can be used either for reading or writing
 type Pin struct {
-	Number    uint
-	direction direction
-	f         *os.File
+	Number     uint
+	direction  Direction
+	logicLevel LogicLevel
+	f          *os.File
 }
 
 // NewInput opens the given pin number for reading. The number provided should be the pin number known by the kernel
-func NewInput(p uint) Pin {
+func NewInput(p uint, logicLevel LogicLevel) Pin {
```

Comment:
I'm fairly hesitant to change this signature. I think the common case is served by the default logic level. What would you think about either a setter on an existing open pin or a new ctor that takes a `LogicLevel`?